openbas-framework/src/main/java/io/openbas/config/OpenBASConfig.java

```diff
@@ -64,8 +67,20 @@ public class OpenBASConfig {
   @JsonIgnore
   private boolean cookieSecure = false;
 
-
   public String getBaseUrl() {
-    return baseUrl.endsWith("/") ? baseUrl.substring(0, baseUrl.length() - 1) : baseUrl;
+    return url(baseUrl);
+  }
+
+  @JsonProperty("application_agent_url")
+  private String agentUrl;
+
```

Comment:
We have to add this on properties file? or we set on settings platforms page?

Reply:
In the application-dev.properties yes

Reply:
Did you test it ?